js/src/redux/providers/signerMiddleware.js

```diff
@@ -82,3 +117,15 @@ export default class SignerMiddleware {
       });
   }
 }
+
+function asHex (val) {
```

Comment:
To keep the code DRY, it would be great if this was part of `/api/format/input` -> `inHex`.